examples/verify-common.sh

```diff
@@ -89,7 +89,7 @@ responds_with () {
     local expected
     expected="$1"
     shift
-    _curl "${@}" | grep "$expected" || {
+    _curl "${@}" | grep -E "$expected" || {
```

Comment:
i think i would prefer not using the `responds_with` function and leaving this as is - rather capture the output and check that it is as expected

Reply:
Done.